src/createNumbersGenerator.js

```diff
@@ -26,7 +26,25 @@
  * @return {number}
  */
 function createNumbersGenerator(numbers, min, max) {
-  // write code here
+  let count = 0;
+  return () => {
+    if (numbers === undefined) {
+      return undefined;
+    } else {
+      for (let i = 0; i < numbers.length; i++) {
+        if (min !== undefined && max !== undefined) {
```

Comment:
good, but if we will receive only `max` or `min`?

you can solve this problem with another check or add default values to min and max